src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -305,7 +333,56 @@ public void onClick(View v) {
         /// initialize block to be moved to single Fragment to retrieve and validate credentials 
         initAuthorizationPreFragment(savedInstanceState);
 
-        //Log_OC.e(TAG,  "onCreate end");
+        mCustomTabPackageName = getCustomTabPackageName();
+
+        if(mCustomTabPackageName != null) {
+            CustomTabsClient.bindCustomTabsService(this, mCustomTabPackageName, mCustomTabServiceConnection);
+        }
+    }
+
+    @Override
+    public void onNewIntent(Intent intent) {
+        super.onNewIntent(intent);
+        // check if its a oauth SALS callback intent

```

Comment:
What is the meaning of SALS here?

Reply:
I added this because I thought SALS would work the same way. But now the are handled completely different, so we can remove it.